code/drasil-example/glassbr/lib/Drasil/GlassBR/Unitals.hs

```diff
@@ -236,13 +236,19 @@ loadDur     = uc' "loadDur"    (nounPhraseSP "duration of load")
   (S "the amount of time that a load is applied to the glass plate")
   (sub lT lDur) Real second
 
-sdx         = uc' "sdx" (nounPhraseSent $ phrase standOffDist +:+ sParen (phrase xComp))
+sdx         = uc' "sdx" 
+  (cn "stand off distance (x-component)")
+  -- (nounPhraseSent $ phrase standOffDist +:+ sParen (phrase xComp))
```

Comment:
I think we can erase the commented out code? Or were you going to keep it for when we try to add back parentheses?

Reply:
I think a TODO or something similar would work; then even if you don't get to it, someone eventually will (and won't delete the useful commented-out code to "clean things up")